modules/crop/architecture.md

```diff
@@ -0,0 +1,50 @@
+# Architecture
+
+Diagram shows basic architecture of Crop API.
+
+![Crop API architecture](images/architecture.png)
+
+## Components
+
+Entity
+------
+- **Crop** Is a content entity that stores various information related to a crop area. It serves to store and returned all informations corresponding to crop area.
+- **CropType** It has a kind bunlde "CropType" who will inherit all the characteristics of the entity mother "Crop" but also allow a variation of Crop entity and will add the information management bind to a type of crop in particular.
```

Comment:
This is not entirely true. Crop type is bundle for Crop entity. It is as simple as that.